packages/ui-library/src/components/form-caption/index.stories.ts

```diff
@@ -44,12 +45,20 @@ export default {
       },
     },
 
+    showIcon: {
+      description: 'Choose if component has an icon.',
+      defaultValue: true,
+      control: { type: 'boolean' },
+      table: { category: 'Content / Settings' },
+    },
+
```

Comment:
As discussed, the showIcon prop should not have a function in the form caption anymore and therefor can be removed here